docs/validation.md

```diff
@@ -0,0 +1,117 @@
+# Validation plugin
+
+This plugin is used with form element to validate user input using ajax.
+
+# Options
```

Comment:
I'd prefer if we could have only one first-level heading per page, and not jump straight from h1 to h3. What do we want the sections of this page to be, what are the sub-sections of each section, etc?